src/main/java/dao/FruitDaoImpl.java

```diff
@@ -0,0 +1,15 @@
+package dao;
+
+import db.Storage;
+
+public class FruitDaoImpl implements FruitDao {
+    @Override
+    public void add(String fruitName, Integer quantity) {
+        Storage.fruits.put(fruitName, quantity);
+    }
+
+    @Override
+    public int get(String fruitName) {
+        return Storage.fruits.get(fruitName);
```

Comment:
Directly accessing the map can lead to a NullPointerException if the fruit is not present. Use `getOrDefault` or check if the fruit exists in the map before getting its value.